docs: max-lines for code editor (#185)
frappe/frappe_docs: frappe_docs/www/docs/user/en/api/controls.md

	// for syntax highlighting
	options: 'Javascript' // JS, HTML, CSS, Markdown, SCSS, JSON,
	// wrap code
	wrap: true,

> 	// max-height of the editor would depend on the number of lines of code if this is set

Reviewer: This and the next comment are unclear. if I set `max_lines` to 5, and there's 3 lines populated, what would be the `max-height`?

Author: It is max lines, so unless you have lines more than that the height would be equivalent to the number of lines.
If you have also set min-lines, the height would always stay between the min-lines and max lines
These are boounds

Reviewer: Documentation is supposed to be simple. If I didn't understand your comment here, how do you expect another read to understand it? The comment is too wordy, and so is your reply; "of the editor would depend on the number of lines of code if this is set" can be condensed a lot IMO.

```suggestion
	// changing `max_lines` will set the max-height of the editor
```

Maybe ^ or something that directly states what is happening. 

In other terms, make docs DRY lol

> 	max_lines: 10

Reviewer: ```suggestion
	max_lines: 10,
```
